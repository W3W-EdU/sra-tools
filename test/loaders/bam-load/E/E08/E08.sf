#bam-load error test #08 : a pair of alignments is missing the LAST flag

r:type=random,name=R1,length=5000

# write all references into one file as FASTA
ref-out:E08.fasta

# write the final SAM-output into this file
sam-out:E08.sam

# write a config-file for bam-load
config:E08.config

p:name=A1,pos=100
p:name=A1,pos=300,tag=T1
t:name=T1,action=clr,flag=LAST

#some additional ( correct ) alignments
p:name=B,repeat=50
p:name=B,repeat=50
