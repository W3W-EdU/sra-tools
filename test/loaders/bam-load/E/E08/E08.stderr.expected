warn: Spots without fragment info have been encountered
warn: libs/vdb/phys-cmn.c:733:VPhysicalProdColumnIdRange: data inconsistent while reading file within application support module - Spot 'A1', which was first seen with mate info, now has no mate info
